virtualizarr/tests/test_readers/test_dmrpp.py

```diff
@@ -320,36 +348,44 @@ def test_parse_filters(basic_dmrpp, var_path, dtype, expected_filters):
 
 
 @pytest.mark.parametrize(
-    "var_path, chunk_shape, expected_lengths, expected_offsets, expected_paths",
+    "var_path, chunk_shape, chunk_grid_shape, expected_lengths, expected_offsets",
     [
         (
             "/data",
             (360, 720),
+            (3, 3),
             np.full((3, 3), 4083, dtype=np.uint64),
             (np.arange(9, dtype=np.uint64) * 4083 + 40762).reshape(3, 3),
-            np.full((3, 3), "test.dmrpp", dtype=np.dtypes.StringDType),
         ),
         (
             "/mask",
             (720, 1440),
+            (1,),
             np.array([4], dtype=np.uint64),
             np.array([41276], dtype=np.uint64),
-            np.array(["test.dmrpp"], dtype=np.dtypes.StringDType),
         ),
     ],
 )
 def test_parse_chunks(
-    basic_dmrpp,
+    tmp_path,
```

Comment:
This test has the same kind of structure as the `open_virtual_dataset` tests I want to write in #243 - because both input test data (the parser) and the expected results need to know about the `tmp_path`.